test/e2e/pvutil.go

```diff
@@ -117,10 +118,10 @@ func deletePersistentVolumeClaim(c clientset.Interface, pvcName string, ns strin
 	}
 }
 
-// Delete the PVC and wait for the PV to become Available again. Validate that the PV
+// Delete the PVC and wait for the PV to enter its expected phase. Validate that the PV
 // has recycled (assumption here about reclaimPolicy). Caller tells this func which
```

Comment:
I'd delete the "recycle" comment. Simply validating that the pv attains its expected Phase.